Client/src/components/common/undermaintenancepage/undermaintenancepage.jsx

```
@@ -0,0 +1,78 @@
+import React from "react";
+import PropTypes from 'prop-types';
+import { Construction, ChevronLeft, ArrowRight } from "lucide-react";
+import { useNavigate } from 'react-router-dom';
+
+const PageUnderConstruction = ({ pageName = "This page" }) => {
+  const navigate = useNavigate();
+
+  return (
+    <div className="bg-gray-900 min-h-screen flex flex-col items-center justify-center relative overflow-hidden">
+      {/* Background elements - matching your home page style */}
+      <div className="absolute top-10 -left-40 w-96 h-96 bg-blue-700 opacity-20 rounded-full filter blur-3xl"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)